rpc/src/v1/impls/light/parity.rs

```diff
@@ -105,13 +105,10 @@ impl ParityClient {
 impl Parity for ParityClient {
 	type Metadata = Metadata;
 
-	fn accounts_info(&self, dapp: Trailing<DappId>) -> Result<BTreeMap<H160, AccountInfo>> {
-		let dapp = dapp.unwrap_or_default();
-
+	fn accounts_info(&self) -> Result<BTreeMap<H160, AccountInfo>> {
```

Comment:
Should we accept (and ignore) the trailing param for backward compatibility?

Reply:
The issue is that we removed several methods, so it's already breaking backward compatibility no matter what. Would it still be useful to keep backward compatibility just for this function?